app/validators/flood_risk_engine/name_format_validator.rb

```diff
@@ -0,0 +1,18 @@
+# Validates the Format of a Name to match the criteria :
+#     Make sure the name contains only letters, spaces, commas, full stops, hyphens and apostrophes
+#
+module FloodRiskEngine
+  class NameFormatValidator < ActiveModel::EachValidator
+
+    def validate_each(record, attribute, value)
+      if(value !~ NameFormatValidator.valid_name_regex)
+        record.errors.add attribute,  I18n.t("flood_risk_engine.validation_errors.name.invalid")
+      end
+    end
+
+    def self.valid_name_regex
+      /\A[\p{Alpha} ,\.'-]+\z/i
```

Comment:
I prefer to stay consistent with WEX and afaik Alpha was originally chosen as your version would fail for e.g Björk Guðmundsdóttir